tests/self/ConvergenceTest.h

```diff
@@ -111,12 +111,11 @@ void runConvergenceTest(
 	    // std::cout << "relativeAccuracy: " << testSet.relativeAccuracy << "\n";
 	    // std::cout << "expectedError: " << testSet.expectedErrors[j][i] << "\n";
 	    logger(INFO, "error: %", error);
-	    logger(INFO, "ExpectedError: %", testSet.expectedErrors[j][i]);
-	    logger(INFO, "ExpectedAccuracy: %", testSet.expectedErrors[j][i]);
-	    
-	    error = testSet.expectedErrors[j][i];
+            if (i < testSet.expectedErrors[j].size()) {
```

Comment:
This was just to not get stuck in the first test, to see if I could see the pattern. This is to be removed.

Reply:
Ok no problem, just had to add it because it started to go out of bounds while testing.